man/dockerd.8.md

```diff
@@ -379,7 +379,8 @@ unix://[/path/to/socket] to use.
   remapping for the user namespace mappings used for contained processes.
```

Comment:
Well, except that if you click the accordion thingy to see context, just two lines up you see 'Specifying "default" will cause a new user and group to be created', which is incorrect for RHEL.